notificationservice.cpp

```diff
@@ -104,8 +103,8 @@ NotificationService *NotificationService::getService() {
     return service;
 }
 
-const QList<NotificationData> &NotificationService::history() const {
-    return _history;
-}
+//const QList<NotificationData> &NotificationService::history() const {
```

Comment:
You need to create your History Model inner this class and create a method that will provide access to the history model using property